Cargo.toml

```diff
@@ -22,13 +22,13 @@ name = "broadcast_bench"
 [features]
 
 [dependencies]
-event-listener = "2.5.1"
-futures-core = "0.3.14"
-easy-parallel = "3.1.0"
-parking_lot = "0.11.2"
+event-listener = "2.5.2"
+futures-core = "0.3.21"
+easy-parallel = "3.2.0"
+parking_lot = "0.12.1"
 
 [dev-dependencies]
-criterion = "0.3"
+criterion = "0.3.5"
 doc-comment = "0.3.3"
-futures-lite = "1.11.3"
-futures-util = "0.3.14"
+futures-lite = "1.12.0"
+futures-util = "0.3.21"
```

Comment:
quite a few unrelated changes. Could you please split the commit to have one logical change per commit, with its own rationale? Thanks.